.github/workflows/release.yml

```diff
@@ -29,4 +29,4 @@ jobs:
           version: latest
           args: release --rm-dist
         env:
-          GITHUB_TOKEN: ${{ secrets.github_token }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Secret names are case sensitive so I'm hoping this was the issue with homebrew publish.